src/ApplicationCore/Entities/BasketAggregate/Basket.cs

```diff
@@ -26,6 +27,7 @@ public void AddItem(int catalogItemId, decimal unitPrice, int quantity = 1)
             return;
         }
         var existingItem = Items.FirstOrDefault(i => i.CatalogItemId == catalogItemId);
+        Guard.Against.Null(existingItem, nameof(existingItem));

```

Comment:
Would changing the previous line to use `First()` instead of `FirstOrDefault` work just as well with one less line of code?